.github/workflows/build_and_test.yml

```diff
@@ -91,6 +91,49 @@ jobs:
         cmake --build . --config Debug
         ctest -C Debug --output-on-failure
 
+  simdUbuntu:
+    name: "SIMD programming using C++ intrinsic functions on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-22.04
+        cxx:
+          - clang++
+    env:
+      ORC_USER_SIMD_LEVEL: AVX512
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: "Test"
+      run: |
+        mkdir -p ~/.m2
+        mkdir build
+        cd build
+        cmake -DBUILD_JAVA=OFF -DBUILD_ENABLE_AVX512=ON ..
+        make package test-out
+
+  simdWindows:
```

Comment:
The downside is that we don't know whether the windows build will have AVX512 support or not. We should at least make sure AVX512-disabled code path is always covered.

Reply:
Hi @dongjoon-hyun @wgtmac , I have already integrated SIMD CI test with the original windows CI test.
https://github.com/wpleonardo/orc/blob/f6b28da5b5fa4eab3917513a7309c0f6a5d96af2/.github/workflows/build_and_test.yml#L76
Currently, if SIMD CI test running on the machine that doesn't support AVX512, it will change to the default path without AVX512. So CI testcase should be always passed.